src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
+@import "./utils/variables.scss";
+
+@import "./blocks/calendar.scss";
+
 body {
```

Comment:
also move the body property to a single file in utils and call it 'resets' for example to reset the margins and paddings and leave the main file only for importing the scss files without any extra thing inside

Reply:
Done!